diff --git a/node_modules/app-builder-lib/out/winPackager.js b/node_modules/app-builder-lib/out/winPackager.js
index 2e68fb5..9a04b03 100644
--- a/node_modules/app-builder-lib/out/winPackager.js
+++ b/node_modules/app-builder-lib/out/winPackager.js
@@ -225,7 +225,10 @@ class WinPackager extends platformPackager_1.PlatformPackager {
             if (this.shouldSignFile(file)) {
                 const parentDir = path.dirname(file);
                 if (parentDir !== packContext.appOutDir) {
-                    return new builder_util_1.CopyFileTransformer(file => this.signIf(file));
+                    return new builder_util_1.CopyFileTransformer(file => {
+                        this.signingQueue = this.signingQueue.then(() => this.signIf(file));
+                        return this.signingQueue;
+                    });
                 }
             }
             return null;
